cas-server-webapp/src/main/webapp/WEB-INF/cas.properties

```diff
@@ -781,3 +781,19 @@ accept.authn.users=casuser::Mellon
 # cas.mfa.request.parameter=authn_method
 # cas.mfa.failure.mode=PHANTOM
 # cas.mfa.authn.ctx.attribute=authnContextClass
+
+#CAS components mappings
+primaryAuthenticationHandler=acceptUsersAuthenticationHandler
+primaryPrincipalResolver=personDirectoryPrincipalResolver
```

Comment:
For OAuth, you don't need any principal resolver, the authentication handler already returns a `HandlerResult`